### DISCUSSION TITLE
Cannot run ASR:  error deprecated numpy  np.complex

### DISCUSSION THREAD

**Discussion by dgoryeo** [PaddlePaddle/PaddleSpeech#3539] [Q&A]
Hi,

I have tried several clean intsllations but I cannot pass this error (please see below for full error output).  It seems that librosa is requiring a version of numy that is not supported. I have tried to upgrade librosa but I get into other incompatiblity between paddlepaddle and paddlespeech. 

Any idea how to resolve this problem please: 

CLI command: '$ paddlespeech asr --lang ja --input ja.wav '

Error output:

`
/paddle/lib/python3.8/site-packages/setuptools/sandbox.py:13: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
/paddle/lib/python3.8/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('mpl_toolkits')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
/paddle/lib/python3.8/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('google')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
Traceback (most recent call last):
  File "/paddle/bin/paddlespeech", line 8, in <module>
    sys.exit(_execute())
  File "/paddle/lib/python3.8/site-packages/paddlespeech/cli/entry.py", line 40, in _execute
    exec("from {} import {}".format(module, cls))
  File "<string>", line 1, in <module>
  File "/paddle/lib/python3.8/site-packages/paddlespeech/cli/asr/__init__.py", line 14, in <module>
    from .infer import ASRExecutor
  File "/paddle/lib/python3.8/site-packages/paddlespeech/cli/asr/infer.py", line 24, in <module>
    import librosa
  File "/paddle/lib/python3.8/site-packages/librosa/__init__.py", line 211, in <module>
    from . import core
  File "/paddle/lib/python3.8/site-packages/librosa/core/__init__.py", line 9, in <module>
    from .constantq import *  # pylint: disable=wildcard-import
  File "/paddle/lib/python3.8/site-packages/librosa/core/constantq.py", line 1059, in <module>
    dtype=np.complex,
  File "/paddle/lib/python3.8/site-packages/numpy/__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'complex'.
`np.complex` was a deprecated alias for the builtin `complex`. To avoid this error in existing code, use `complex` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.complex128` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
` 

Thanks

**Comment by halley100** [comment]
i faced the same problem, have you fixed that yet?
